### PR TITLE
Fixed improper navigation semantics in accessibility contexts

### LINKED ISSUE #197: Improper navigation semantics
### Bug Report

|    Q        |   A
|------------ | ------
| Version(s)  | `^2.8`

#### Summary

When a link to the currently displayed page is rendered, it should include the `aria-current="page"` attribute.

#### Current behavior

Navigation links are not providing "current page" semantics in the accessibility tree.

#### How to reproduce

Using the screen reader of your choice, activate a menu link, then bring focus back to that same a menu element.

#### Expected behavior

Varies based on assistive technology being used. As an example, VoiceOver macOS will read a properly tagged navigation link as "current page, visited, link, Example Page".

### PR DESCRIPTION
**Pull request by ivyrze**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | no
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | yes

### Description

Adds an `aria-current="page"` to active menu links and breadcrumb segments, resolving #197.

The menu tests as previously designed have more than one page marked as active, which means `aria-current` is being used improperly in those test scenarios. I'm assuming that in real-world use, this would not be the case?

### CODE REVIEW

File: src/Helper/Navigation/Breadcrumbs.php [hunk 1]
```diff
@@ -111,9 +111,13 @@ public function renderStraight($container = null)
         } else {
             /** @var View\Helper\EscapeHtml $escaper */
             $escaper = $this->view->plugin('escapeHtml');
-            $html    = $escaper(
+            $label   = $escaper(
```
Comment (Ocramius) [inline src/Helper/Navigation/Breadcrumbs.php:114]: The type seems valid here: re-generating the Psalm baseline should suffice. Not sure why `mixed` is inferred, but your code is good as-is :+1:

File: src/Helper/Navigation/Breadcrumbs.php [hunk 2]
```diff
@@ -111,9 +111,13 @@ public function renderStraight($container = null)
...
                 $this->translate($active->getLabel(), $active->getTextDomain())
             );
+            $attribs = [
+                'aria-current' => 'page',
+            ];
+            $html    = '<span' . $this->htmlAttribs($attribs) . '>' . $label . '</span>';
```
Comment (Ocramius) [inline src/Helper/Navigation/Breadcrumbs.php:120]: Similar to above: re-generating the psalm baseline will suffice

**Review by Ocramius — APPROVED** [review]
Besides the small Psalm misalignment, this LGTM! 

Thanks @ivyrze. Give the baseline a try, and if it fails, I'll pick it from here, just let me know.

**Comment by ivyrze** [comment]
Thanks for the pointer @Ocramius!

**Review by Ocramius — APPROVED** [review]
Thanks @ivyrze!

**Pull request merged**